masterlist.yaml

```diff
@@ -2637,6 +2637,42 @@ plugins:
     clean:
       - crc: 0xAA9D9F35
         util: 'SSEEdit v3.2.44'
+  - name: 'WheelsOfLull.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/748'
+        name: 'The Wheels Of Lull on Nexus Mods'
+    clean:
+      - crc: 0xC191A3F9 # v1.3a, unofficial patch
+        util: 'SSEEdit v3.3.4 BETA'
+    msg:
+      - type: warn
+        content: 'Update Patch available: %1%'
+        subs: [ '[Wheels of Lull Unofficial Patch](https://www.nexusmods.com/skyrimspecialedition/mods/19734).' ]
+        condition: 'checksum("WheelsOfLull.esp", FF400946) or checksum("WheelsOfLull.esp", 6771AF29)' # first is before, second is after cleaning
```

Comment:
```suggestion
        # Before & after cleaning.
        condition: 'checksum("WheelsOfLull.esp", FF400946) or checksum("WheelsOfLull.esp", 6771AF29)'
```

Reply:
> Make as many silent comments as you want within the masterlists - it doesn't hurt (within reason). However, make sure to put them above the line(s) that they refer to.

Minor thing, but just for consistency. But not something I'd hold up a pull over. Rest looks good.